packages/dashboard/package.json

```diff
@@ -40,6 +40,7 @@
     "react-router-dom": "^6.0.0-beta.0",
     "react-use": "^17.3.1",
     "unstated-next": "^1.1.0",
+    "urlcat": "^2.0.4",
```

Comment:
maybe we need to move it to the root package.json

Reply:
vote +1

Reply:
Only two packages are needed the `urlcat` at this time. And the root package.json is very minimalist.

Reply:
Don't move it to the top. It's the correct way to use pnpm.